target/scripts/build/packages.sh

```diff
@@ -82,12 +77,17 @@ function _install_packages() {
     opendmarc libsasl2-modules sasl2-bin
   )
 
+  DEBUG_PACKAGES=(
+    bind9-dnsutils iproute2 iputils-ping less nano
+  )
```

Comment:
Is `iproute2` for the `ip` command?

Should we add some clarification here for `bind9-dnsutils` choice?:

```suggestion
  # `bind9-dnsutils` provides `dig` command
  # `iproute2` provides `ip` command
  DEBUG_PACKAGES=(
    bind9-dnsutils iproute2 iputils-ping less nano
  )
```

Reply:
> Is `iproute2` for the `ip` command?

Yes and it's mandatory and not just for debugging. We use the `ip` command in [networking.sh](https://github.com/docker-mailserver/docker-mailserver/blob/894978ddd751f0d09e9f68e1a3de13926e17d432/target/scripts/startup/setup.d/networking.sh#L16).

Reply:
Right - I will remove it from `DEBUG_PACKAGES` and add it back to the normal set.

Reply:
Done